tests/services/inventory/test_data/additional_cai_access_policies.dump

```diff
@@ -0,0 +1,7 @@
+{"name":"//cloudresourcemanager.googleapis.com/organizations/111222333","asset_type":"cloudresourcemanager.googleapis.com/Organization","access_policy":{"name":"accessPolicies/111222333444","parent":"organizations/abc","title":"default policy"},"ancestors":["organizations/111222333"]}
```

Comment:
I dont think this file is being used.

Reply:
Removed file. Mock files are being used.